backend/app/tests/api/routes/test_creds.py

```
@@ -188,274 +94,197 @@ def test_update_credentials(
-        f"{settings.API_V1_STR}/credentials/", headers={"X-API-KEY": user_api_key.key}
-    )
-    response = client.delete(
-        f"{settings.API_V1_STR}/credentials/provider/{Provider.OPENAI.value}",
-        headers={"X-API-KEY": user_api_key.key},
-    )
+    """Verify credentials are encrypted in database."""
+    db_credential = db.exec(
+        select(Credential).where(
+            Credential.organization_id == user_api_key.organization_id,
+            Credential.project_id == user_api_key.project_id,
+            Credential.is_active == True,
```

Ruff (0.14.10)

Avoid equality comparisons to `True`; use `Credential.is_active:` for truth checks

Replace with `Credential.is_active`

(E712)